docs/Superoperator representations.md

```diff
@@ -12,49 +16,565 @@ where $i$ is a row and $j$ is a column index.
 
 We define `vec` to be column stacking
```

Comment:
It would be worth pointing out that this corresponds to how matrices are stored in memory for column major storage conventions. It may resonate with numerics folks.

Reply:
done, see below.